static/css/friend.css

```diff
@@ -0,0 +1,130 @@
+#article-container {
+	word-wrap: break-word;
+	overflow-wrap: break-word;
+}
+
+#article-container a {
+	color: #49b1f5;
+}
+
+#article-container a:hover {
+	text-decoration: underline;
+}
+
+#article-container img {
+	margin: 0 auto 0.8rem;
+}
```

Comment:
Is `article-container` a good name for friend lists? You also have a `flink` class which seems to refer to the same container. Maybe we could just use a single `friend-container` class and refer to the container as `div.friend-container`? (Or choose some other appropriate names.)

Moreover, is it really necessary to have both the container and the `friend-list-div` or just 1 container would be enough for your use case?